js/renovation/ui/scheduler/workspaces/base/date_table/all_day_panel/layout.tsx

```diff
@@ -0,0 +1,59 @@
+import {
+  Component, ComponentBindings, JSXComponent, OneWay, Fragment,
+} from 'devextreme-generator/component_declaration/common';
+import { Table } from '../../table';
+import { AllDayPanelTableBody as TableBody } from './table_body';
+import { addHeightToStyle } from '../../../utils';
+import { GroupedViewData, ViewCellData } from '../../../types.d';
+import { AllDayPanelTitle as Title } from './title';
+
+export const viewFunction = (viewModel: AllDayPanelLayout) => (
+  <Fragment>
```

Comment:
Can we return null when `isVisbile` is false instead of using Fragment?

Reply:
Unfortunately no